Why and when to use HTML forms (#27)
pupilfirst/web-development-101-script: building-web-forms/why-html-forms/README.md

# Text

> Forms in HTML are a very powerful tool for interacting with users — most commonly they are used for collecting data from users, or allowing them to control a web application.

Reviewer: ```suggestion
Forms in HTML are a very powerful tool for interacting with the users — most commonly they are used for collecting data, or allowing users to control a web application.
```
